File: src/main/java/br/com/color4rit/model/Mapa.java
package br.com.color4rit.model;

import br.com.color4rit.enums.Cor;
import br.com.color4rit.enums.Dificuldade;

public class Mapa {

    private Long id;

    private Dificuldade dificuldade;

    private Musica musica;


    public Musica getMusica() {
        return musica;
    }

    public void setMusica(Musica musica) {
        this.musica = musica;
    }

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }



    @Override
    public String toString() {
        return "Mapa{" +
                "id=" + id +
                ", dificuldade=" + dificuldade +
                ", musica=" + musica +
                '}';
    }
}

File: src/main/webapp/js/Mapa.js
import Musica from './Musica.js'

class Mapa {
    constructor(id, dificuldade, musica) {
        this.id = id,
        this.dificuldade = dificuldade,
        this.musica = new Musica(musica.autor, musica.nome, musica.duracao)
    }
};

export default Mapa;